Error TS2305: Module '"react-element-to-jsx-string"' has no exported member 'Options'

_**storybookjs/addon-jsx**_ fails when updating from `14.1.0` to `14.3.0` with the error above, because the following line:

```tsx
import reactElementToJSXString, { Options } from 'react-element-to-jsx-string';
```

I'm not familiar with publishing types, but I believe the error is because related to [index.d.ts](https://github.com/algolia/react-element-to-jsx-string/blob/master/index.d.ts) not exporting `ReactElementToJSXStringOptions`, perhaps as `Options`?

The build fails with  the error above: 

```
src/index.tsx:4:35 - error TS2305: Module '"react-element-to-jsx-string"' has no exported member 'Options'.

4 import reactElementToJSXString, { Options } from 'react-element-to-jsx-string';
```

My wild guess is that flow exported it implicitly, because it was part of the signature of the default function exported. 

For TS, this seems to work:

```diff
diff --git a/index.d.ts b/index.d.ts
index c2bc46a..83ff4c6 100644
--- a/index.d.ts
+++ b/index.d.ts
@@ -13,7 +13,8 @@ declare module 'react-element-to-jsx-string' {
     sortProps?: boolean;
     useFragmentShortSyntax?: boolean;
   }
+  export { ReactElementToJSXStringOptions as Options };

   const reactElementToJSXString: (element: ReactNode, options?: ReactElementToJSXStringOptions) => string;
-  export = reactElementToJSXString;
+  export default reactElementToJSXString;
 }
```

I was able to use the new filterProps by linking (`yarn link`) this project and building _**storybook-addon-jsx**_.

--- a/index.d.ts
+++ b/index.d.ts
@@ -1,9 +1,11 @@
 declare module 'react-element-to-jsx-string' {
   import { ReactNode } from 'react';
 
+  type FilterPropsFunction = (value: any, key: string) => boolean
+
   interface ReactElementToJSXStringOptions {
     displayName?: (element: ReactNode) => string;
-    filterProps?: string[];
+    filterProps?: string[] | FilterPropsFunction;
     showDefaultProps?: boolean;
     showFunctions?: boolean;
     functionValue?: (fn: any) => any;
@@ -15,5 +17,8 @@ declare module 'react-element-to-jsx-string' {
   }
 
   const reactElementToJSXString: (element: ReactNode, options?: ReactElementToJSXStringOptions) => string;
-  export = reactElementToJSXString;
+
+  export { ReactElementToJSXStringOptions as Options };
+
+  export default reactElementToJSXString;
 }

